### ISSUE TITLE
Add numbering scheme for releases and ontologies

### ISSUE THREAD

**Issue by harryhoch** [monarch-initiative/monarch-legacy#42]
Each release should have a unique identifier. About pages should have appropriate ID #s for ontologies releases.

**Comment by mellybelly** [comment]
For ontologies, we have have a few choices. Some ontologies in the import will have access dates, others release IRIs. We can make available the full monarch.owl import chain as part of the provenance of the release. Or we can somehow list for every import the access date/release IRI. or do both. Can Ontoquest provide this info?

For the data, I like the above approach.

This all could be included in some kind of report page, that also includes the stats and/or graph of data in the system- aka Nicole's nice graphs showing phenotype X organism coverage. Basically we need a system status page to include all this?

**Comment by mellybelly** [comment]
Yes, and ideally we'd have a synopsis of a) what is in the system and b) what is available for semantic similarity?

**Comment by cmungall** [comment]
We have this, mostly for debugging:
$BASEURL/admin/introspect

OWL Sim
http://owlsim.monarchinitiative.org/

We can have any kind of display we want of these. However, I am currently unaware of a way to introspect globally metadata for all resources.

**Comment by mellybelly** [comment]
related to #299

**Comment by larrybabb** [comment]
Is there a status on this issue? As integrators of the monarch toolset, this is becoming a more critical issue for our users.

**Comment by jmcmurry** [comment]
The data itself is now archived [here](https://archive.monarchinitiative.org/). I will leave it to @cmungall and @TomConlin to comment on the versions of the ontologies to which the data is annotated, and the versions of the ontologies used to perform the integration and drive the platform.

**Comment by larrybabb** [comment]
That is helpful, but I have a follow up...
Essentially, I am looking for a stable and accurate representation of the data and its updates over time.

So should I go to the Releases page or the https://archive.monarchinitiative.org/ page to get an accurate record of the version of data over time? 
The archive page shows more changes than the Releases page.
I'd also like to know if there are ever changes that occur to the underlying data that may not be contained in published release notes and/or archive lists?

**Comment by jmcmurry** [comment]
We endeavor to keep the releases page (https://monarchinitiative.org/page/releases) up to date. There are some intermediate data releases in the archive  https://archive.monarchinitiative.org/  that may not necessarily have appeared on the UI/dumps/API. We need to be doing a much better job of making it clearer what is authoritative and corresponds to what distribution of the data. Asking @TomConlin and @kshefchek to comment by way of this comment: are data dumps and API services perfectly synchronous? What is the path to making this airtight?
